src/Domain.LinnApps/Reports/GoodsInLogReportService.cs

```diff
@@ -33,23 +33,24 @@ public ResultsModel GoodsInLogReport(
             string storagePlace)
         {
             var fromDateSearch = string.IsNullOrWhiteSpace(fromDate)
-                               ? DateTime.Now.AddDays(-14)
+                               ? (DateTime?)null
```

Comment:
does this mean it will just pull back every result in existence if you dont specify any dates?